script/cmtool.bat

```diff
@@ -98,10 +98,10 @@ goto exit0
 :puppet
 ::::::::::::
 
-:: if "%CM_VERSION%" == "latest" set CM_VERSION=3.8.7
+if "%CM_VERSION%" == "latest" set CM_VERSION=6.3.0
 
-if not defined PUPPET_URL set PUPPET_64_URL=https://downloads.puppetlabs.com/windows/puppet-x64-%CM_VERSION%.msi
-if not defined PUPPET_URL set PUPPET_64_URL=https://downloads.puppetlabs.com/windows/puppet-%CM_VERSION%.msi
+if not defined PUPPET_URL set PUPPET_64_URL=https://downloads.puppetlabs.com/windows/puppet/puppet-agent-%CM_VERSION%-x64.msi
```

Comment:
Not sure if you're interested, but if entirely possible (?) I'd like to support the older URL scheme for the prior versions of puppet as this scheme only works for the 6.0-series.

If it turns out that we need to do tricky shit to do major version comparisons, then at least we could expose a `puppet1` CM or something similar.